mmv1/templates/terraform/examples/dataplex_task_notebook.tf.erb

```diff
@@ -0,0 +1,60 @@
+# VPC network
+resource "google_compute_network" "default" {
+    name                    = "tf-test-workstation-cluster%{random_suffix}"
+    auto_create_subnetworks = true
+}
+
+
+data "google_project" "project" {
+
+}
+
+resource "google_dataplex_lake" "<%= ctx[:primary_resource_id] %>" {
+  name         = "tf-test-lake%{random_suffix}"
+  location     = "us-central1"
+  project = "<%= ctx[:test_env_vars]['project_name'] %>"
+}
+
+
+resource "google_dataplex_task" "<%= ctx[:primary_resource_id] %>" {
+
+    task_id      = "tf-test-task%{random_suffix}"
+    location     = "us-central1"
+    lake         = google_dataplex_lake.<%= ctx[:primary_resource_id] %>.name
+    trigger_spec  {
+        type = "RECURRING"
+        schedule = "1 * * * *"
+    }
+    
+    execution_spec {
+        service_account = "${data.google_project.project.number}-compute@developer.gserviceaccount.com"
+        args = {
+            TASK_ARGS  = "--output_location,gs://spark-job-jars-anrajitha/task-result, --output_format, json"
+        }
+    }
+    notebook {
+        notebook = "gs://ssmadhavik-test4/Demo/CarPricePrediction_OLS_ML.ipynb"
```

Comment:
will the tests fail if this is not pointing to a real URI? if the resource allows for it, this should ideally be genericized to not point to an individual's personal test url like that.